docs/best_practice/data_import.txt

```diff
@@ -228,6 +228,9 @@ COPY FROM Command
   cr> REFRESH TABLE user;
   REFRESH OK (... sec)
 
+  cr> delete from user;
```

Comment:
no I guess it's okay.. but is the refresh then really needed if it is deleted immediately anyway?